### DISCUSSION TITLE
Add a new attribute to Desktop

### DISCUSSION THREAD

**Discussion by soliton37** [ansys/pyaedt#5295] [Ideas]
Hello,
Would it be possible to add a Desktop.pid attribute to retrieve the process ID session that is written in the pyaedt log?
At the moment, I am looking for this information in the log file.
Thanks, Emmauel.

**Comment by gmalinve** [comment]
Hi @soliton37,

You could use the native API: oDesktop.GetProcessID().

Thanks,

Giulia

**Reply by Samuelopez-ansys** [reply]
Also the using pyaedt syntax: aedtapp.desktop_class.aedt_process_id

**Comment by soliton37** [comment]
Thank for support
